### DISCUSSION TITLE
teable as a Photography library in Postgres

### DISCUSSION THREAD

**Discussion by edoardoc** [teableio/teable#1057] [Ideas]
Hello there, 

I was thinking of using this to manage my photography library, uploading photos as LO's into PostgreSQL and then organize them into tags, etc. with teable

**Comment by tea-artist** [comment]
<img width="1538" alt="image" src="https://github.com/user-attachments/assets/766a1c2b-2c30-47fb-bc24-8eb2c5eb4c91">
Of course, the new gallery view is perfect for your needs.

**Reply by edoardoc** [reply]
WOW thank you!
just a quick set of questions :)

are the pics stored as LO's?
is the EXIFF interpreted into db fields?
please if you could point me to some docs about this

**Reply by tea-artist** [reply]
Pictures are stored in the volume as files, and EXIFF is not interpreted
